pending/fontconfig_mod1.xml

```diff
@@ -31,10 +31,13 @@ along with this program. If not, see <http://www.gnu.org/licenses/>.
     <action command="delete" search="glob" path="~/fontconfig/*.cache-?"/>
     <action command="delete" search="glob" path="~/.cache/fontconfig/*"/>
     <action command="delete" search="glob" path="/var/cache/fontconfig/*"/>
+    <!-- Win XP -->
+    <action command="delete" search="glob" path="$LOCALAPPDATA\fontconfig\cache\*.cache-?"/>
```

Comment:
Thank you for adding Windows XP. Because fontconfig is not common on Windows, for future reference it would be useful to document which application you used to test this.